components/cdc/src/endpoint.rs

```diff
@@ -1086,6 +1093,21 @@ impl Initializer {
 
         fail_point!("cdc_incremental_scan_start");
 
+        let deregister_region = || {
```

Comment:
Should `deregister_region` be called before L1150

Reply:
Technically yes, but if the program ever gets to L1150, there is likely to be a very serious congestion or deadlock, and deregistering is likely to fail as well.